### PR TITLE
PR: Source terminal config files to have conda in PATH

### LINKED ISSUE #164: Spyder terminal does not activate conda environments and keeps asking to "conda init" even after initialization.
## Description of your problem
I use Spyder as part of the Anaconda distribution (which limits the version of Spyder to be 3.3.6). I used conda to install spyder-terminal. First problem is that when I fire up a new terminal window in Spyder, the base conda environment doesn't seem to have been activated with (base) prefix even though "which python" will return the correct path. Then when I try to activate any conda environment (including base), I get a CommandNotFound error asking to use conda init. Even after using "conda init bash" or "conda init --all" and restarting the shell, the CommandNotFound error persists.

**What steps will reproduce the problem?**

1.Start a new terminal.
2. conda activate my_env
3. conda init --all
4. Restart shell
5. conda activate my_env

**What is the expected output? What do you see instead?**
I expect to see my_env activated while both after step #2 and step #5, I see this error message:

_CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'._

**Please provide any additional information below**

## Versions and main components

* Terminal Version: native terminal Version 2.8.2
* Spyder Version: 3.3.6
* Python Version: 3.7
* Operating system: Mac OS 10.13.6

### LINKED ISSUE #250: Shell interpreter: Anaconda Power Shell Prompt
## Description of proposed enhancement

**Anaconda pre-activate shell interpreter**
image

I am running spyder-ide from a new created environment (not conda-base). I would like to access anaconda commands within syder-terminal, but the two shells proposed do not include "Anaconda Power Shell Prompt" or "Anaconda Prompt".

[In Atom or for batch commands I usually type "call C:\Users\mario\Anaconda3\Scripts\activate.bat" first in the cmd to activate Anaconda.](https://github.com/issues)

## Versions and main components

* Terminal Version: 0.5.0
* Spyder Version: 4.2.0
* Python Version: 3.8.5
* Operating system: Windows 10

### PR DESCRIPTION
**Pull request by steff456**
This PR adds conda support to the terminal,

image

It was tested on all platforms and running Spyder without the development mode.

Fixes #164 
Fixes #250

### CODE REVIEW

File: spyder_terminal/widgets/terminalgui.py [hunk 1]
```diff
@@ -66,6 +66,20 @@ def close(self):
...
+        "sh": ["~/.profile", "~/.shrc", "~/.shinit"],
+        "ksh": ["~/.profile", "~/.kshrc"],
+        "csh": ["~/.cshrc", "~/.login"],
+        "pwsh": [],
+        "rbash": ["~/.bashrc", "~/.bash_profile"],
+        "dash": ["~/.profile"],
+        "screen": [],
+        "tmux": [],
+        "tcsh": ["~/.tcshrc"],
+        "xonsh": ["~/.xonshrc"]}
```
Comment (ccordoba12) [inline spyder_terminal/widgets/terminalgui.py:82]:
```suggestion
        "xonsh": ["~/.xonshrc"]
    }
```

File: spyder_terminal/widgets/terminalgui.py [hunk 2]
```diff
@@ -108,6 +122,15 @@ def setup_term(self):
             dict_options[option] = self.get_conf(option)
         self.apply_settings(dict_options)
         self.apply_zoom()
+        shell_name = self.get_conf('shell')
+        if os.name != 'nt':
```
Comment (ccordoba12) [inline spyder_terminal/widgets/terminalgui.py:126]: What happens for Windows? Can envs be activated without problems there?

**Review by ccordoba12 — COMMENTED** [review]
Thanks a lot for this @steff456! It's pretty nice!

Also, please add a test for this so we can catch regressions in the future. Testing with Bash on Linux and Zsh on macOS would be fine.

File: spyder_terminal/widgets/terminalgui.py (hunk 2 above)
Comment (steff456) [reply spyder_terminal/widgets/terminalgui.py:126]: Yes, they work as it is

File: spyder_terminal/widgets/terminalgui.py [hunk 3]
```diff
@@ -66,6 +66,21 @@ def close(self):
...
+        "ksh": ["~/.profile", "~/.kshrc"],
+        "csh": ["~/.cshrc", "~/.login"],
+        "pwsh": [],
+        "rbash": ["~/.bashrc", "~/.bash_profile"],
+        "dash": ["~/.profile"],
+        "screen": [],
+        "tmux": [],
+        "tcsh": ["~/.tcshrc"],
+        "xonsh": ["~/.xonshrc"]
+        }
```
Comment (ccordoba12) [inline spyder_terminal/widgets/terminalgui.py:83]:
```suggestion
    }
```

**Review by ccordoba12 — COMMENTED** [review]
@steff456, a couple of more for you then this should be ready.

**Review by ccordoba12 — APPROVED** [review]
Great work here, thanks a lot @steff456!

**Pull request merged**